turbot/table_turbot_policy_value.go

```diff
@@ -0,0 +1,211 @@
+package turbot
+
+import (
+	"context"
+	"fmt"
+	"strconv"
+
+	"github.com/turbot/steampipe-plugin-sdk/grpc/proto"
+	"github.com/turbot/steampipe-plugin-sdk/plugin"
+	"github.com/turbot/steampipe-plugin-sdk/plugin/transform"
+)
+
+func tableTurbotPolicyValue(ctx context.Context) *plugin.Table {
+	return &plugin.Table{
+		Name:        "turbot_policy_value",
+		Description: "Policy value define the value of policy known to Turbot.",
+		List: &plugin.ListConfig{
+			Hydrate: listPolicyValue,
+			KeyColumns: []*plugin.KeyColumn{
+				{Name: "state", Require: plugin.Optional},
+				{Name: "policy_type_id", Require: plugin.Optional},
+				{Name: "resource_id", Require: plugin.Optional},
+				{Name: "resource_type_id", Require: plugin.Optional},
+				{Name: "filter", Require: plugin.Optional},
+			},
+		},
+		Columns: []*plugin.Column{
+			// Top columns
+			{Name: "id", Type: proto.ColumnType_INT, Transform: transform.FromField("Turbot.ID"), Description: "Unique identifier of the policy value."},
+			{Name: "policy_type_title", Type: proto.ColumnType_STRING, Transform: transform.FromField("Type.Title"), Description: "Title of the policy type."},
+			{Name: "poliy_type_trunk_title", Type: proto.ColumnType_STRING, Transform: transform.FromField("Type.Trunk.Title"), Description: "Title with full path of the policy type."},
+			{Name: "is_default", Type: proto.ColumnType_BOOL, Transform: transform.FromField("Default"), Description: "Defines the policy value is default or not."},
+			{Name: "is_calculated", Type: proto.ColumnType_BOOL, Description: "True if this is a policy setting will be calculated for each value."},
```

Comment:
Similar to `is_default`, and other columns in this table, the description does not have correct grammar, what description does our GraphQL API have?

Reply:
We have description for is_calculated is `If true this value is derived from calculated setting inputs e.g. templateInput and template.` in GraphQL API.

Reply:
@ParthaI Let's copy the descriptions, they read OK in a description context here, and being able to re-use them is an easy way to get accurate, grammatically correct descriptions.

Reply:
Updated the description for `is_default` and `is_calculated` columns